### PR TITLE
Cleanup GROUPING SETS formating

### PR DESCRIPTION
**Pull request by novikd**
Changelog category (leave one):

- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: tests/queries/0_stateless/01883_with_grouping_sets.reference [hunk 1]
```diff
@@ -73,6 +73,19 @@
...
 0	0	49500
+SELECT
+    fact_1_id,
+    fact_3_id,
+    sum(sales_value) AS sales_value
+FROM grouping_sets
+GROUP BY
+    GROUPING SETS (
+        (fact_1_id),
+        (fact_1_id, fact_3_id))
```
Comment (azat) [inline tests/queries/0_stateless/01883_with_grouping_sets.reference:84]: What do you think about improving the formatting with one element?

Right now (including this changes) it includes extra whitespace:

```sql
SELECT *
FROM data
GROUP BY
    GROUPING SETS ( (foo))
```

Either this:

```sql
SELECT *
FROM data
GROUP BY
    GROUPING SETS ((foo))
```

Or this:

```sql
SELECT *
FROM data
GROUP BY
    GROUPING SETS (
        (foo)
    )
```

Comment (alexey-milovidov) [reply tests/queries/0_stateless/01883_with_grouping_sets.reference:84]: Thank you! I also cannot stand extra whitespace :)

Comment (alexey-milovidov) [reply tests/queries/0_stateless/01883_with_grouping_sets.reference:84]: Second example is ok.

**Comment by novikd** [comment]
@Mergifyio update

**Comment by alexey-milovidov** [comment]
Looks like there are some defects in GROUPING SETS implementation?

**Comment by novikd** [comment]
@Mergifyio rebase

**Comment by novikd** [comment]
@Mergifyio update

**Pull request merged**